src/Renderer/WebXR.js

```diff
@@ -0,0 +1,72 @@
+/* global XRRigidTransform */
+
+import * as THREE from 'three';
+import { VRButton } from 'ThreeExtended/webxr/VRButton';
+
+async function shutdownXR(session) {
+    if (session) {
+        await session.end();
+    }
+}
+
+const initializeWebXR = (view, options) => {
+    document.body.appendChild(VRButton.createButton(view.mainLoop.gfxEngine.renderer));
```

Comment:
I am not at all familiar with how VR/AR works on browsers, but shouldn't `VRButton` be appended to `view.domElement` instead for cases when an iTowns viewer is included inside some other content ? I don't know if this is possible in VR/AR though.

Reply:
Fixed, appended to the `view.domElement`.